src/librekey/rnp_key_store.cpp

```diff
@@ -438,11 +424,11 @@ rnp_key_store_free(rnp_key_store_t *keyring)
    \return none
 */
 bool
-rnp_key_store_list(pgp_io_t *io, const rnp_key_store_t *keyring, const int psigs)
+rnp_key_store_list(const rnp_key_store_t *keyring, const int psigs)
 {
     unsigned keyc = (keyring != NULL) ? list_length(keyring->keys) : 0;
 
-    (void) fprintf(io->res, "%u key%s\n", keyc, (keyc == 1) ? "" : "s");
+    (void) fprintf(stdout, "%u key%s\n", keyc, (keyc == 1) ? "" : "s");
```

Comment:
Maybe this function should take a `FILE` (or `std::ostream`?) which accepts the output, instead of it always being printed to stdout?

Reply:
Yeah, makes sense. Added argument and will pass rnp->resfp there.